src/hooks/gatewayAuth/useTaskAuthorization.ts

```diff
@@ -56,7 +56,7 @@ const useTaskAuthorization = (prefixes: string[]) => {
         grants.find((grant) => prefix.startsWith(grant))
     );
 
-    if (!hasLength(authorizedPrefixes)) {
+    if (!hasLength(authorizedPrefixes) && hasLength(prefixes)) {
```

Comment:
These errors sometimes spam a lot - so adding an extra check for when the page is loading and there are no `prefixes`